README.md

```diff
@@ -1,14 +1,68 @@
-# Cosmos
+<p align="center">
 
-[![Build status](https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true)](https://ci.appveyor.com/project/CosmosOS/cosmos/branch/master)
-[![Join the chat at (https://discord.com/invite/kwtBwv6jhD](https://img.shields.io/discord/833970409337913344)](https://discord.com/invite/kwtBwv6jhD)
+  <img src="https://user-images.githubusercontent.com/63316499/89792973-43587480-daf3-11ea-99d6-82f89dd2ffc3.png" width="25%" />
 
-Cosmos is an operating system "construction kit" made for Visual Studio 2022. Build your own OS using managed languages such as C#, VB.NET, and more!
+</p>
 
-For instructions on how to install and use Cosmos, please visit the [Cosmos website](http://www.gocosmos.org).
+<p align="center">
+
+  <a href="https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master">
+    <img src="https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true" />
+  </a>
+
+  <img src="https://img.shields.io/github/languages/code-size/CosmosOS/Cosmos" />
+  <img src="https://img.shields.io/github/downloads/CosmosOS/Cosmos/total" />
+
+  <a href="https://github.com/CosmosOS/Cosmos/releases/latest">
+    <img src="https://img.shields.io/github/v/release/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://github.com/CosmosOS/Cosmos/blob/master/LICENSE.txt">
+    <img src="https://img.shields.io/github/license/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://github.com/CosmosOS/Cosmos/issues">
+    <img src="https://img.shields.io/github/issues/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://discord.com/invite/kwtBwv6jhD">
+    <img src="https://img.shields.io/discord/833970409337913344" />
+  </a>
+
+</p>
+
+<hr/>
+
+Cosmos (C# Open Source Managed Operating System) is an operating system development kit which uses .NET, alongside the custom IL2CPU compiler to convert (most) C# code into a working bare-metal OS.
+Despite C# in the name, any .NET-based language can be used, which includes: VB.NET, IronPython, F# and more. Cosmos itself and its kernel routines are primarily written in C#, and thus the Cosmos name.
 
+In a project, Cosmos can be thought of as a compiler and a sort-of standard library for a project. It gives the user access to often hard to find or otherwise difficult to understand tools.
+
+<hr/>
+
+## Features
+
+The following is a non-exhaustive list of features that Cosmos offers:
+
+- Low level assembly access and pointer memory control
+- A basic (and unstable at the moment) filesystem
+- Most features found in the .NET core library
+- A CPU/FPU accelerated math library
+- A basic graphics interface
+- A basic network interface
+- A basic audio interface
+
+> **Note**
+
```

Comment:
I think it looks better if we remove this newline

Reply:
> I think it looks better if we remove this newline

It breaks without it, it tries to merge both lines into one